asap/em_montage_qc/detect_montage_defects.py

```diff
@@ -163,6 +169,21 @@ def detect_stitching_gaps(render, prestitched_stack, poststitched_stack,
     session.close()
     return gap_tiles
 
+def detect_distortion(render, poststitched_stack, zvalue, threshold_cutoff=[0.005, 0.005], pool_size=20):
+    z_to_scales = {zvalue: do_get_z_scales_nopm(zvalue, [poststitched_stack], render)}
+    
+    # check if any scale is None
+    zs = [z for z, scales in z_to_scales.items() if scales is None]
```

Comment:
I don't think this is necessary, and the above line can just raise an error if it hits one by using `get_z_scales_nopm`

Reply:
code changed to adapt to catch the error and remove unwanted lines of code.